cmd/alias-import.go

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2015-2022 MinIO, Inc.
+//
+// This file is part of MinIO Object Storage stack
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package cmd
+
+import (
+	"encoding/json"
+	"os"
+	"strings"
+
+	"github.com/minio/mc/pkg/probe"
+
+	"github.com/minio/cli"
+)
+
+var aliasImportCmd = cli.Command{
+	Name:      "import",
+	ShortName: "i",
+	Usage:     "Import configuration info to configuration file from a JSON formatted string ",
```

Comment:
```suggestion
	Usage:     "import configuration info to configuration file from a JSON formatted string ",
```
starts lower cased by convention